File: Api-BackEnd/api/src/main/java/br/com/aikodigital/api/models/Equipment.java
package br.com.aikodigital.api.models;

import java.util.UUID;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "equipment")
@Getter
@Setter
public class Equipment {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    private UUID id;
    
    @Column(name = "name")
    private String name;

    @ManyToOne
    @JoinColumn(name = "equipment_model_id", referencedColumnName = "id")
    private Equipment_Model equipment_model_id;

    
    
}

File: EquipmentDomain/Models/Equipment.cs
using System;
using System.Collections.Generic;

namespace EquipmentDomain.Models;

public partial class Equipment
{
    public Guid Id { get; set; }

    public Guid EquipmentModelId { get; set; }

    public string Name { get; set; } = null!;

    public virtual EquipmentModel EquipmentModel { get; set; } = null!;
}
